src/Discord.Net.Core/Entities/Users/IPresence.cs

```diff
@@ -13,5 +15,9 @@ public interface IPresence
         ///     Gets the current status of this user.
         /// </summary>
         UserStatus Status { get; }
+        /// <summary>
+        ///     Gets the set of clients where this user is currently active.
+        /// </summary>
+        IImmutableSet<ClientType> ActiveClients { get; }
```

Comment:
I left a `TODO: Un-comment ActiveClients definition in IPresence for 2.1.x breaking update` comment. I'm assuming that a breaking revision this small wouldn't require a major release, given that I'm not sure it would affect much other than addons, maybe?

Reply:
Wait till C#8 then we could throw a default interface implementation and it wouldn't be breaking anymore to add to the interface.

Reply:
Is there any way we could use preprocessor directives for the C# version?

Reply:
> Wait till C#8 then we could throw a default interface implementation and it wouldn't be breaking anymore to add to the interface.

That would force us to target only .NET Core instead of .NET Standard.